### PR TITLE
Dependency tree support

### PR DESCRIPTION
**Pull request by nemanjan00**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/bazilio91/ejs-compiled-loader",
   "dependencies": {
-    "ejs": "^1.0.0",
+    "ejs": "git+https://github.com/nemanjan00/ejs.git",
```
Comment (bazilio91) [inline package.json:26]: Pin to commit sha please.

**Comment by nemanjan00** [comment]
Sorry, I do not understand what you said... What should I do?

**Comment by bazilio91** [comment]
It's not very secure to use master of git repo as a default. Please specify commit ish in packages.json https://docs.npmjs.com/files/package.json#git-urls-as-dependencies

**Comment by nemanjan00** [comment]
Can you merge it now?

**Comment by nemanjan00** [comment]
Since they do not plan to make any changes to ejs, is it ok to just leave my fork of ejs as dependency?

**Pull request merged**